src/index.js

```diff
@@ -8,8 +9,23 @@ const defaultOptions = {
   source: true,
 };
 
+const defaultMtrcConf = {
+  h1: H1,
+  h2: H2,
+  h3: H3,
+  h4: H4,
+  h5: H5,
+  h6: H6,
+  code: Code,
+  p: P,
+  a: A,
+  li: LI,
+  ul: UL,
+};
+
 export default {
-  addWithInfo(storyName, info, storyFn, _options) {
+  addWithInfo(storyName, info, storyFn, _options, _mtrcConf) {
```

Comment:
Hi @rook91 
Yes, there's no point of having a `mtrcConf` argument when the `info` argument is missing but if we just consider the code without that assumption it's a bug.
I'll merge this PR now.